libraries/hatch-server-user-management/src/AWSCognitoClient.ts

```diff
@@ -224,4 +243,40 @@ export default class AWSCognitoClient implements UserServiceClient {
     return pemCerts;
   }
   
+  private convertAWSErrorToUserManagementError(awsError: AWSError) {
+    switch (awsError.code) {
+      case 'CodeMismatchException' || 'ExpiredCodeException': {
+        return new UserManagementError(UserManagementErrorCodes.INVALID_CODE, awsError.message);
+      }
+      case 'InvalidPasswordException': {
```

Comment:
Remove unnecessary braces wrapping each case. If/when we add ESLint complexity rules, this is prob going to need to be a lookup in a dictionary.

Reply:
updated